src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.DefaultOutputPaths.targets

```diff
@@ -1,25 +0,0 @@
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Given that @adalon & @kzu *added* this file in 13d216fd86b4419eee8029abecf2ef24115edf26, we might want them to chem in on it's *removal*...

Reply:
Since the purpose of this PR is to improve the way the incremental clean works, I'm fine having the previous mechanism to improve its perf/reliability be removed too. I suspect changing the target framework / API level for a project isn't that common anyway (maybe less so even in the .NET5 way some day...)